cal/cal/views.py

```diff
@@ -126,6 +126,7 @@ def google_auth(request):
         storage.put(credential)
         profile, _ = Profile.get_or_create(user=request.user)
         profile.authed = True
+        profile.first_time_authed = True
```

Comment:
I actually don't think you need the `first_time_authed` field. If you run incremental sync every time the user logs in it will already handle all the behavior you need. In the case where it's the user's first time logging in ever, inc sync will try to get a sync token, fail, and then run a full sync. In the normal case, when it's not the user's first time logging in, inc sync will work fine (after we fix it of course).